app-android/src/main/java/io/github/droidkaigi/confsched2023/license/License.kt

```diff
@@ -0,0 +1,21 @@
+package io.github.droidkaigi.confsched2023.license
+
+import kotlinx.collections.immutable.PersistentList
+import kotlinx.collections.immutable.persistentListOf
+
+data class License(
```

Comment:
Could you put sources to appropriate modules?
I think 
UI will be feature/about, 
Repository's implementation and api will be core/data,
This(License) will be core/model.

Reply:
That's make sense.

`OssLicenseRepositoryImpl` is still in the app module since limitation of Google's license plugin.
Except that, placing to appropriate modules. 🙇